CMakeLists.txt

```diff
@@ -149,6 +149,12 @@ if(NOT INSTRUCTIONS_PATH)
   message(STATUS "INSTRUCTIONS_PATH set to ${INSTRUCTIONS_PATH}")
 endif()
 
+# RevCPU syscalls path
+if(NOT SYSCALLS_PATH)
+  set(SYSCALLS_PATH "${CMAKE_CURRENT_SOURCE_DIR}/include/syscalls")
+  message(STATUS "SYSCALLS_PATH set to ${SYSCALLS_PATH}")
+endif()
+
```

Comment:
This will backout the changes I pushed to devel yesterday

Reply:
> This will backout the changes I pushed to devel yesterday

Again, a merge anomaly. Not sure why.